policies/PrivacyPolicy.html

```diff
@@ -469,8 +470,61 @@ <h3><strong>Information automatically collected</strong></h3>
                                           </ul>
                                           <div><h2></h2></div>
                                           <div><br /></div>
+                                          <div id="thirdpartysources">
+                                            <strong><h2>5. WHEN AND HOW DO WE RECEIVE INFORMATION FROM THIRD PARTIES?</h2></strong>
+                                          </div>
+                                          <div><br /></div>
+                                          <div>
+                                            <strong><em>In Short:</em></strong><em> We may receive information from third parties from time to time, for example, in order to fulfill moderation tickets.</em> 
+                                          </div>
+                                          <div></div>
+                                          <div><br /></div>
+                                          <div><strong>Users, Vendors, and Partners.</strong> Data may be shared with us by users of our products and services, vendors, and partners ("<strong>third party sources</strong>") who utilize the platform.  We may at times accept this information as it pertains to the following activities as executed by Yellow Dog Man Studios, its Contractors, and its Partners:</div>
```

Comment:
I dont understand this inclusion. All of these are covered by other factors. Why was it added?